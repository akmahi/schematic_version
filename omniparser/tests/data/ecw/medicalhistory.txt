apuid|RecordID|PatientId|EncounterId|EncounterDate|ProviderNPI|PatientAccountNo|PastHistory|HistoryICDs
305922|305922.534105|534105|678578|2015-04-20|1841223997|||
305922|305922.11360|11360|705342|2014-06-10|1194808725|||
305922|305922.533222|533222|705261|2015-02-24|1205940038|||
305922|305922.535266|535266|705365|2011-12-02|1629117296|||
305922|305922.533015|533015|705258|2015-11-16||||
305922|305922.533151|533151|705260|2014-10-22|1417179151|||
305922|305922.533796|533796|705262|2010-10-27||||
305922|305922.534115|534115|705266|2015-02-25|1841223997|||
305922|305922.531775|531775|705275|2014-01-24||||
305922|305922.9609|9609|705281|2011-10-06|1225022072|||
305922|305922.533118|533118|705273|2013-09-19|1225022072|||
305922|305922.533380|533380|705276|2015-04-13|1467551002|||
305922|305922.533303|533303|705280|2015-03-27|1437290483|||
305922|305922.532939|532939|705278|2014-05-14||||
305922|305922.11430|11430|705279|2012-06-19||||
305922|305922.533288|533288|705339|2015-06-19|1891804704|||
305922|305922.533782|533782|705340|2014-05-22|1780671446|||
305922|305922.533652|533652|705341|2012-05-29|1780671446|||
305922|305922.532901|532901|698842|2016-02-15|1841377272|||
305922|305922.532962|532962|703707|2013-04-09||||
305922|305922.533235|533235|703830|2010-05-24||||
305922|305922.529691|529691|702823|2016-05-03|1386731784|||
305922|305922.533304|533304|704912|2015-07-24||||
305922|305922.533200|533200|704914|2014-10-29||||
305922|305922.532938|532938|704917|2013-11-06||||
305922|305922.533087|533087|704919|2010-08-05|1225022072|||
305922|305922.533007|533007|704920|2013-03-26|1235244211|||
305922|305922.534184|534184|704299|2013-09-17|1164505715|||
305922|305922.531100|531100|704300|2013-08-30||||
