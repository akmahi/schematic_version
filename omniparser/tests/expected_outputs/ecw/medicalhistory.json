[
	{
		"medical_history": {
			"encounter": {
				"date": "2015-04-20",
				"id": "678578"
			},
			"patient": {
				"id": "534105"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1841223997"
			},
			"record_id": "305922.534105"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2014-06-10",
				"id": "705342"
			},
			"patient": {
				"id": "11360"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1194808725"
			},
			"record_id": "305922.11360"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2015-02-24",
				"id": "705261"
			},
			"patient": {
				"id": "533222"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1205940038"
			},
			"record_id": "305922.533222"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2011-12-02",
				"id": "705365"
			},
			"patient": {
				"id": "535266"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1629117296"
			},
			"record_id": "305922.535266"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2015-11-16",
				"id": "705258"
			},
			"patient": {
				"id": "533015"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533015"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2014-10-22",
				"id": "705260"
			},
			"patient": {
				"id": "533151"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1417179151"
			},
			"record_id": "305922.533151"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2010-10-27",
				"id": "705262"
			},
			"patient": {
				"id": "533796"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533796"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2015-02-25",
				"id": "705266"
			},
			"patient": {
				"id": "534115"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1841223997"
			},
			"record_id": "305922.534115"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2014-01-24",
				"id": "705275"
			},
			"patient": {
				"id": "531775"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.531775"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2011-10-06",
				"id": "705281"
			},
			"patient": {
				"id": "9609"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1225022072"
			},
			"record_id": "305922.9609"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2013-09-19",
				"id": "705273"
			},
			"patient": {
				"id": "533118"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1225022072"
			},
			"record_id": "305922.533118"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2015-04-13",
				"id": "705276"
			},
			"patient": {
				"id": "533380"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1467551002"
			},
			"record_id": "305922.533380"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2015-03-27",
				"id": "705280"
			},
			"patient": {
				"id": "533303"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1437290483"
			},
			"record_id": "305922.533303"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2014-05-14",
				"id": "705278"
			},
			"patient": {
				"id": "532939"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.532939"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2012-06-19",
				"id": "705279"
			},
			"patient": {
				"id": "11430"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.11430"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2015-06-19",
				"id": "705339"
			},
			"patient": {
				"id": "533288"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1891804704"
			},
			"record_id": "305922.533288"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2014-05-22",
				"id": "705340"
			},
			"patient": {
				"id": "533782"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1780671446"
			},
			"record_id": "305922.533782"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2012-05-29",
				"id": "705341"
			},
			"patient": {
				"id": "533652"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1780671446"
			},
			"record_id": "305922.533652"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2016-02-15",
				"id": "698842"
			},
			"patient": {
				"id": "532901"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1841377272"
			},
			"record_id": "305922.532901"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2013-04-09",
				"id": "703707"
			},
			"patient": {
				"id": "532962"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.532962"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2010-05-24",
				"id": "703830"
			},
			"patient": {
				"id": "533235"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533235"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2016-05-03",
				"id": "702823"
			},
			"patient": {
				"id": "529691"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1386731784"
			},
			"record_id": "305922.529691"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2015-07-24",
				"id": "704912"
			},
			"patient": {
				"id": "533304"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533304"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2014-10-29",
				"id": "704914"
			},
			"patient": {
				"id": "533200"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.533200"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2013-11-06",
				"id": "704917"
			},
			"patient": {
				"id": "532938"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.532938"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2010-08-05",
				"id": "704919"
			},
			"patient": {
				"id": "533087"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1225022072"
			},
			"record_id": "305922.533087"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2013-03-26",
				"id": "704920"
			},
			"patient": {
				"id": "533007"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1235244211"
			},
			"record_id": "305922.533007"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2013-09-17",
				"id": "704299"
			},
			"patient": {
				"id": "534184"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1164505715"
			},
			"record_id": "305922.534184"
		}
	},
	{
		"medical_history": {
			"encounter": {
				"date": "2013-08-30",
				"id": "704300"
			},
			"patient": {
				"id": "531100"
			},
			"practice": {
				"apuid": "305922"
			},
			"record_id": "305922.531100"
		}
	}
]
